PubMed.js

```diff
@@ -36,33 +36,32 @@
  */
  
   
-  /*****************************
+/** ***************************
```

Comment:
I assume that's the work of `lint --fix`? any idea why -- doesn't make a lot of sense to me (although of course also doesn't hurt). (And thanks for doing this @zuphilip  !)

Reply:
There is a rule ([spaced-comment](https://eslint.org/docs/rules/spaced-comment)) which enforces a space after `//` or `/*` of a comment. The `/**` seems also be okay as start of jsdoc.

There is an example how to keep such commenting blocks with a lot of asterisk:
```
/* eslint spaced-comment: ["error", "always", { "exceptions": ["*"] }] */

/****************
 * Comment block
 ****************/
```
Should we change to that in the configuration of ESLint?

Reply:
I would say so, yes. Commenting blocks are nice